modules/smaatoBidAdapter.md

```diff
@@ -63,6 +63,50 @@ var adUnits = [{
 }];
 ```
 
+For native adunits:
+
+```
+var adUnits = [{
+    "code": "native unit",
+    "mediaTypes": {
+        native: {
```

Comment:
not sure if this one should stay the same, but I guess we will see it once we submit it to upstream

Reply:
or based on the spec test it should have `nativeOrtbRequest` here, shouldn't it?

Reply:
for the test we need to use the new version if we use it

when a publisher configures the adunits he (as of now) can choose to either use new approach or legacy approach
there is "native.js" in prebid.js that translates legacy to new behind the scenes, so we'll receive also new one in both cases. but that isn't used for the tests.

official examples all still show adunits, so i didn't touch it for now